pkg/instancegroups/instancegroups.go

```diff
@@ -65,6 +67,32 @@ func NewRollingUpdateInstanceGroup(cloud fi.Cloud, cloudGroup *cloudinstances.Cl
 	}, nil
 }
 
+// User input routine copied from vendor/google.golang.org/api/examples/gmail.go
+func PromptInteractive(upgradedHost string) (stop_prompting bool) {
```

Comment:
GoLang likes `stopPrompting` instead of `stop_prompting` :)

Reply:
Also `promptInteractive` instead, no need to make it public, and  https://blog.golang.org/godoc-documenting-go-code

The tldr; on the documentation is.

1. start with the func name. `promptInteractive`
2. write a sentence and end it with a period.

Reply:
How about `func promptInteractive(upgradedHost string) (stopPrompting bool, err error)`? Then you would be able to return the error up the stack.